packages/appstore/src/components/modals/wallets-migration-failed/wallets-migration-failed.tsx

```diff
@@ -22,28 +18,16 @@ const WalletsMigrationFailed = observer(() => {
         setWalletsMigrationFailedPopup(false);
     };
 
-    const validateClickOutside = (e: MouseEvent) => {
```

Comment:
@sergei-deriv why did you remove this?

Reply:
@mahdiyeh-deriv I've deleted this because Modal component already has `validateClickOutside` and `useOnClickOutside` functions and we just don't need to `set has_close_icon = false` and it works by default.